topo/topo.go

```diff
@@ -98,6 +100,12 @@ func WithBasePath(s string) Option {
 	}
 }
 
+func WithProgress(b bool) Option {
```

Comment:
Please ad a godoc comment.  (All exported functions, variables, and types should have a godoc comment).